src/page-building/career/perks.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Teaser } from '../../components/teasers/teaser';
+import { Illustration } from '../../components/illustration/illustration';
+import { up } from '../../components/breakpoint/breakpoint';
+import { IllustrationType } from '../../components/illustration/illustration-set';
+import { SectionHeader } from '../../new-components/section-header/section-header';
+
+interface Perk {
+  illustration: IllustrationType;
+}
+
+const TeaserGrid = styled.div`
```

Comment:
I didn't want to articially share the grid. It's local here and once we refactor and decide for some central location of this grid I'm happy to remove this duplicated code (as it exists in other sections too)